ce_discovery.yaml

```diff
@@ -114,32 +80,91 @@ paths:
     delete:
       operationId: deleteService
       parameters:
-       - in: path
-         name: id
-         description: The id of the service to be returned
-         required: true
-         schema:
-           $ref: '#/components/schemas/service/properties/id'
+        - $ref: '#/components/parameters/id'
       requestBody:
-        required: false
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/serviceinstance'
+        $ref: '#/components/requestBodies/servicedeleterequest'
```

Comment:
Having a requestBody on a delete is discouraged by OpenAPI for reasons discussed [here](https://github.com/OAI/OpenAPI-Specification/issues/1801).  To abusively summarize the most useful piece, there is no standard about the interaction of HTTP DELETEs on to resource collections and the caching layers which may cause undesirable lack of caching on one hand and/or a use of cached resources inappropriately on the other.

e.g.

```
DELETE /services

[
  { id: {id_0} },
  ...
  { id: {id_N} },
]
```

Does not clear `/services/{id_0}` from HTTP response caches which, unless configured not to cache, are allowed to cache and return that value following the resource's deletion using a DELETE with a request body.